packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -770,6 +770,126 @@ export class Gtxna extends Op {
   }
 }
 
+// represents branch name of a new branch
+export class Label extends Op {
+  readonly label: string;
+
+    /**
+   * @param args words list extracted from line
+   * @param line line number in TEAL file
+   */
+  constructor (args: string[], line: number) {
+    super();
+    assertLen(args.length, 1, line);
+    this.label = args[0];
```

Comment:
`args` received will be `label:`. store it in `this.label` accordingly

Reply:
Thanks for catching.